libcudacxx/test/libcudacxx/force_include.h

```diff
@@ -17,6 +17,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#define TEST_NVRTC_VIRTUAL_DEFAULT_CTOR_ANNOTATION
```

Comment:
This is the wrong macro name, we are using it for the *destructor* not the *constructor*